src/app/pages/apps/components/installed-apps/installed-apps.component.ts

```diff
@@ -38,33 +43,25 @@ export class InstalledAppsComponent implements OnInit, AfterViewInit {
   isLoading = false;
   filterString = '';
   appJobs = new Map<string, Job<ChartScaleResult, ChartScaleQueryParams>>();
+  routeAppIdForDetails: string;
 
   entityEmptyConf: EmptyConfig = {
     type: EmptyType.Loading,
     large: false,
     title: helptext.message.loading,
   };
 
-  constructor(
-    private appService: ApplicationsService,
-    private cdr: ChangeDetectorRef,
-    private router: Router,
-    private layoutService: LayoutService,
-    private matDialog: MatDialog,
-    private dialogService: DialogService,
-    private snackbar: SnackbarService,
-    private translate: TranslateService,
-    private slideInService: IxSlideInService,
-  ) {
-    this.router.events
-      .pipe(
-        filter((event) => event instanceof NavigationEnd),
-        untilDestroyed(this),
-      )
-      .subscribe(() => {
-        this.layoutService.pageHeaderUpdater$.next(this.pageHeader);
-      });
-  }
+  emptySearchResultsConf: EmptyConfig = {
```

Comment:
It was moved to `showLoadStatus()` method under the [master](https://github.com/truenas/webui/blob/master/src/app/pages/apps/components/installed-apps/installed-apps.component.ts#L162) branch.